Views/General/Feed/Card.swift

```diff
@@ -89,19 +89,21 @@ struct Card: View {
             VStack(alignment: .trailing) {
                 ForEach(trailingPropertyList, id: \.self) { info in
                     Text(info)
-                        .foregroundColor(.white)
+                        .foregroundColor(.accentColor)
+                        .fontWeight(.semibold)
                         .padding(4)
                         .background(
                             RoundedRectangle(cornerRadius: 8)
+                                .fill(.white)
                         )
-                        .foregroundColor(.black)
                         .font(.caption)
                 }
             }
             .padding([.top, .trailing], 10)
         }
-        .overlay(RoundedRectangle(cornerRadius: cornerRadius).stroke(Color.primary.opacity(0.3), lineWidth: 2))
+        .overlay(RoundedRectangle(cornerRadius: cornerRadius).stroke(Color.primary.opacity(0.3), lineWidth: 0))
```

Comment:
wait, this modifier was only supposed to add a rounded-border to the view, if setting lineWidth to 0 we may as well just remove it.